src/game_battle.h

```diff
@@ -31,6 +31,9 @@ namespace RPG {
 }
 
 namespace Game_Battle {
+    /** @return true if a battle is currently running */
+    bool IsBattleRunning();
```

Comment:
wrong whitespace. Also considering that "Init" is the "constructor" it should be somewhere below Init